gazel.go

```diff
@@ -177,6 +178,19 @@ func (v *Venderor) updatePkg(path, _ string, pkg *build.Package) error {
 	if pkg.IsCommand() {
 		rules = append(rules, newRule("go_binary", filepath.Base(pkg.Dir), attrs))
 	} else {
+		if len(cgoSrcs.List) != 0 {
+			cgoPname := "cgo_default_library"
+			cgoDeps := v.extractDeps(pkg.TestImports)
+			cgoRule := newRule("cgo_library", cgoPname, map[string]bzl.Expr{
+				"srcs":      cgoSrcs,
+				"clinkopts": asExpr([]string{"-ldl", "-lz", "-lm", "-lpthread", "-ldl"}),
+			})
+			rules = append(rules, cgoRule)
+			if len(cgoDeps.List) != 0 {
+				cgoRule.SetAttr("deps", cgoDeps)
```

Comment:
I guess i didn't like the empty deps = [] blocks but I don't really care actually. Removed.